src/components/TodoComponents/Todo.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+
+class Todo extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      todo: { task: "" }
+    };
+  }
+  render() {
+    console.log(this.props.list);
+    console.log(this.props.list.task);
+    return (
+      <ul>
+        <li onClick={this.props.onClick}>{this.props.list.task}</li>
```

Comment:
Also When you are clicking the task, that is deleting it instead of changing it to `completed: true` on state. You could keep the delete functionality on a button .